apps/scraper/migrations/0002_auto_20190828_0950_squashed_0004_auto_20190828_0956.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.0.12 on 2019-08-28 09:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    replaces = [('scraper', '0002_auto_20190828_0950'), ('scraper', '0003_auto_20190828_0954'), ('scraper', '0004_auto_20190828_0956')]
```

Comment:
Please, rename this file to '0001_initial.py' and remove this line. See: https://djbook.ru/rel1.8/topics/migrations.html#squashing-migrations